src/sentry/search/events/fields.py

```diff
@@ -433,7 +433,8 @@ def resolve_field_list(
         if "project.id" not in fields:
             fields.append("project.id")
 
-    for field in fields[:]:
+    snuba_condition_aggregates = snuba_filter.condition_aggregates or []
+    for field in fields[:] + snuba_condition_aggregates:
```

Comment:
Should we move this to after where auto_aggregations are added? Something could be in condition_aggregates but `auto_aggregations` might be false (alternatively we could check the boolean here too)

nit: Could we use sets here to avoid potentially iterating on the same columns 2x

Reply:
Agree with the `auto_aggregation` suggestion.